uk.co.compucorp.civicrm.hrleaveandabsences/xml/Menu/import.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+<menu>
+  <item>
+    <path>civicrm/admin/leaveandabsences/import</path>
+    <page_callback>CRM_HRLeaveAndAbsences_Import_Controller</page_callback>
+    <title>Import Leave/Absence Requests</title>
+    <access_arguments>administer leave and absences</access_arguments>
+  </item>
+</menu>
```

Comment:
I only noticed this now. Is there any reason that this is on its own xml instead of in `xml/Menu/hrleaveandabsences.xml`?

Also, we'll need to add a link to this page in the menu. Can you please check with Mike where would be the best place for it?